securesystemslib/keys.py

```diff
@@ -713,9 +716,9 @@ def create_signature(key_dict, data):
     sig, scheme = securesystemslib.ed25519_keys.create_signature(
         public, private, data, scheme)
 
-  # Continue to support keytypes of ecdsa-sha2-nistp256 and ecdsa-sha2-nistp384
+  # Continue to support keytypes of ecdsa-sha2-nistp256
   # for backwards compatibility with older securesystemslib releases
-  elif keytype in ['ecdsa', 'ecdsa-sha2-nistp256', 'ecdsa-sha2-nistp384']:
+  elif keytype in ['ecdsa', 'ecdsa-sha2-nistp256',]:
```

Comment:
If the `keytype` is `ecdsa-sha2-nistp384` the execution will continue with the call of `securesystemslib.ecdsa_keys.create_signature(public, private, data, scheme)`. 
I am wondering what will be the value of the `scheme` on this call?
Because the `scheme` argument is the one used to determine if the `ecdsa-sha2-nistp384` signature generation is actually supported.

If the scheme is again  `ecdsa-sha2-nistp384`, then there is no sense in keeping this comment and check and how to drop support for `ecdsa-sha2-nistp256` and `ecdsa-sha2-nistp384` could be decided independently of this.

Reply:
I decided to not update the commit now until we decide what to do next.

Reply:
Nitpick: I recommend using a set instead of a list to capture the values to check...

Reply:
> If the scheme is again ecdsa-sha2-nistp384, then there is no sense in keeping this comment and check and how to drop support for ecdsa-sha2-nistp256 and ecdsa-sha2-nistp384 could be decided independently of this.

Not sure I follow your reasoning, but I don't think this particular line of code is worth any further consideration without taking into account the big picture in #308 or the even bigger picture in https://github.com/secure-systems-lab/signing-spec/issues/9.

Reply:
> Long story short, instead of making this unrelated change now, I'd rather think about how to best deprecate both 'ecdsa-sha2-nistp256' and 'ecdsa-sha2-nistp384' as keytypes.

Yes please, let's figure out how to deprecate both 'ecdsa-sha2-nistp256' and 'ecdsa-sha2-nistp384' as keytypes.

> @joshuagl, do you want to weigh in as the author of that last change?

With or without this change we don't support generating signatures with a keytype of ecdsa-sha2-nistp384, so I don't feel too strongly about how it fails. It was a mistake to introduce the change in #228, but let's not worry too much about trying to undo it now and focus on the bigger picture items and deprecating the overloaded keytypes.

Reply:
Okay, then I will leave this commit for now.